tensorflow-core/tensorflow-core-api/src/gen/java/org/tensorflow/internal/c_api/Node.java

```diff
@@ -159,5 +159,5 @@ public class Node extends Pointer {
 
   // Called after an incident non-control edge has changed. Does nothing if not
   // all input edges are defined.
-  public native void RunForwardTypeInference();
+  
```

Comment:
Why'd the method disappear but not the comment?

Reply:
I think that's what happen when you skip a method in JavaCPP preset, I had to do this with this one since the symbol was found nowhere in the native libs, see line 327 in `tensorflow.java` patch.